.travis.yml

```diff
@@ -43,7 +43,7 @@ script:
       bash .misc/tests.sh
       python setup.py bdist_wheel
       pip install ./dist/coala-*.whl
-      pip install coala-bears --pre -U
+      pip install coala-bears[alldeps] --pre
```

Comment:
why the removal of `-U`? IMO we should either consistently use the newest (with -U) or spec a version (`==0.10.0dev.....`)